File: cpp/grammer/08_if_else.cpp
#include <iostream>
#include <string>

int main() {
    int score = 82;
    std::string grade;
    if (score >= 90) grade = "A";
    else if (score >= 80) grade = "B";
    else grade = "C";

    std::cout << "점수: " << score << "\n등급: " << grade << "\n";
}

File: python/grammer/08_if_else.py
"""8) 조건문: if / elif / else 기본 형태."""

score = 82

if score >= 90:
    grade = "A"
elif score >= 80:
    grade = "B"
else:
    grade = "C"

print("점수:", score)
print("등급:", grade)
